README.md

```diff
@@ -1,5 +1,29 @@
 # Northcoders News API
 
+## View the app [here](https://obscure-lowlands-69895.herokuapp.com/api)
+
+--
+
+## About
+
+This is a RESTful API for articles, comments and users. It includes multiple endpoints with various params and queries. You can view `/api` to get a list of all available endpoints.
+
```

Comment:
I would look at bulking this part out a bit, what is the api designed to do? What tech have you used?